corehq/toggles.py

```diff
@@ -1810,3 +1810,10 @@ def _enable_bi_integration_preview(domain, enabled):
     TAG_PRODUCT,
     [NAMESPACE_DOMAIN],
 )
+
+SESSION_MIDDLEWARE_LOGGING = StaticToggle(
+    'session_middleware_logging',
+    'Log all session object method calls on this domain',
+    TAG_CUSTOM,
+    [NAMESPACE_DOMAIN]
```

Comment:
Does this middleware happen after we know what user it is? Could we make this a user toggle so that we can restrict it to only test users?

Reply:
This comes prior to the users middleware, so the user object is not available to get the toggle info.